Library/Homebrew/tab.rb

```diff
@@ -250,6 +250,11 @@ def to_s
     when true  then s << "Poured from bottle"
     when false then s << "Built from source"
     end
+    unless time.nil?
+      s << "Installed" if s.empty?
+      s << "on"
+      s << Time.at(time).strftime("%Y-%m-%d")
```

Comment:
I think `s << Time.at(time).strftime("on %Y-%m-%d at %H:%M:%S")` would be better (colons instead of dashes as separators for the time). I'm definitely in favor of adding the time, too.